libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerActionBuilder.cs

```diff
@@ -140,27 +140,27 @@ private async Task<DialogTurnResult> CallGenerateAnswerAsync(WaterfallStepContex
             }
 
             // Calling QnAMaker to get response.
-            var response = await _services.GetAnswersAsync(stepContext.Context, qnaMakerOptions).ConfigureAwait(false);
+            var response = await _services.GetAnswersRawAsync(stepContext.Context, qnaMakerOptions).ConfigureAwait(false);
 
             // Resetting previous query.
             dialogOptions[PreviousQnAId] = -1;
             stepContext.ActiveDialog.State["options"] = dialogOptions;
 
             // Take this value from GetAnswerResponse 
-            var isActiveLearningEnabled = true;
+            var isActiveLearningEnabled = response.ActiveLearningEnabled;
 
-            stepContext.Values[QnAData] = new List<QueryResult>(response);
+            stepContext.Values[QnAData] = new List<QueryResult>(response.Answers);
             
             // Check if active learning is enabled.
-            if (isActiveLearningEnabled)
+            if (isActiveLearningEnabled && response.Answers.Any() && response.Answers.First().Score <= 0.95)
```

Comment:
Should this be configurable?
Magic numbers are always suspicious. Is this a widely accepted QnA parameter?

Reply:
No, It is not configurable. Its upper cap for calling active learning, 
We check if the score is below 95%, then only we consider feedback for that code, else when confidence score is very high (ie above 95%), we don't check for feedbacks.

Reply:
I have modified this const to a private variable.